src/main/webapp/user-page.html

```diff
@@ -20,6 +20,8 @@
     <meta charset="UTF-8">
     <title>User Page</title>
     <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
```

Comment:
This is a question: if you look at line 22 vs. 23, the href's are different. Usually I have a slash before the word "css" so it looks like this: "/css/somefile.css". Does it not work for you if you have that initial slash? I thought we had to have that slash so it can know where the target css file is.

Reply:
ok